plugins/scaffolder-backend/src/scaffolder/actions/builtin/helpers.ts

```diff
@@ -132,6 +132,7 @@ type BranchProtectionOptions = {
   repoName: string;
   logger: Logger;
   requireCodeOwnerReviews: boolean;
+  requiredStatusChecks: string[];
```

Comment:
Think that should should be `requiredStatusChecks?` as you're providing a default value anyway below. Otherwise the type thinks that it's required, and you're never gonna provide a default value.

Reply:
Maybe it should also be `requiredStatusCheckContexts` instead. As we're passing this directly as the context value to the Octokit api.

Reply:
Done.